src/org/javarosa/core/model/instance/TreeElement.java

```diff
@@ -438,6 +437,10 @@ public TreeElement getBindAttribute(String namespace, String name) {
     /**
      * get value of the bind attribute with namespace:name' in the list
      *
+     * Example
```

Comment:
Better to add `@param`s and say that namespace may be null if there is no namespace.